src/commons/api/swagger.json

```diff
@@ -0,0 +1,3361 @@
+{
```

Comment:
This file is copied from cadet. Do you think there should be a yarn command to copy the file over for convenience?